libsolidity/analysis/TypeChecker.cpp

```diff
@@ -3191,6 +3194,19 @@ bool TypeChecker::visit(Identifier const& _identifier)
 	return false;
 }
 
+void TypeChecker::endVisit(IdentifierPath const& _identifierPath)
+{
+	_identifierPath.annotation().requiredLookup =
```

Comment:
Also here - I think we should only use virtual lookup if _identifierPath has only one element.